test/net/sourceforge/kolmafia/AreaCombatDataTest.java

```diff
@@ -87,15 +87,17 @@ public class AreaCombatDataTest {
           Map.entry(MonsterDatabase.findMonster("Danglin' Chad"), 801),
           Map.entry(MonsterDatabase.findMonster("War Frat Streaker"), 901));
 
+  private static final String TESTUSERNAME = "AreaCombatDataTestUser";
+
   @BeforeEach
   public void beforeEach() {
-    KoLCharacter.reset("fakeUserName");
-    Preferences.reset("fakeUsername");
+    KoLCharacter.reset(TESTUSERNAME);
+    Preferences.reset(TESTUSERNAME);
   }
 
   @AfterAll
   public static void deleteQueueFile() {
-    File queueF = new File(KoLConstants.DATA_LOCATION, "fakeusername_queue.ser");
+    File queueF = new File(KoLConstants.DATA_LOCATION, TESTUSERNAME + "_queue.ser");
```

Comment:
previously, the username was lowercased. Now it's not. Desired?

Reply:
I don't expect the casing to matter but to be safe...

Reply:
Changed it.  I suspect it was lowercased because that was the case of the file on the disk when the cleanup was written.

Reply:
Casing matters on non-Windows filesystems. If the casing is wrong, the file won't get deleted.

Reply:
code is
```java
  public static String baseUserName(final String name) {
    return name == null || name.equals("")
        ? "GLOBAL"
        : StringUtilities.globalStringReplace(name.trim(), " ", "_").toLowerCase();
  }
```

Reply:
Thank you.  My original code was definitely an unintended error.